process/process.md

```diff
@@ -0,0 +1,291 @@
+# Preface
+
+This document defines processes adopted by Xen-Troops project. It
+should be followed by all Xen-Troops members. Parts of the document
+that describe commits, pull requests and such should be followed by
+external contributors as well.
+
+# Definition of Done
+
+Every task is considered completed when resulting changes are fully
+integrated into the codebase. This means that you have done your task
+only **after** your Pull Request(s) was/were merged. Before that
+moment your task is **not** done.
+
+Not all tasks end as pull requests. For example, there can be research
+tasks where you research some topic or experiment with different
+approaches. Those still require some tangible results, like Wiki page
+for example.
+
+# Submitting changes
+
+We adhere to processes developed by Linux kernel community. Please
+read ["Submitting
+Patches"](https://www.kernel.org/doc/html/latest/process/submitting-patches.html#separate-your-changes),
+["Patch
+preparation"](https://www.kernel.org/doc/html/latest/process/5.Posting.html#patch-preparation)
+and ["Patch formatting and
+changelogs"](https://www.kernel.org/doc/html/latest/process/5.Posting.html#patch-formatting-and-changelogs)
+
+Also you can check our [contributing
+guide](https://xen-troops.github.io/contribution)
+
+Here we will re-iterate most important points and describe how it
+applies to GitHub, as we don't use mailing-list based review.
+
+## Coding style
+
+Every project has own coding style so we can't provide here any
+specific requirements. But you should always adhere to the coding
+style of a project where you making commit. When in doubt - look at
+surrounding code and follow its style.
+
```

Comment:
Please suggest style to stick to when starting **new project(s)** in C, C++, Rust, Python, Bash. I really hate seeing CaMeLcAsE one day.